crates/api_models/src/admin.rs

```diff
@@ -75,6 +75,9 @@ pub struct MerchantAccountCreate {
     #[cfg(not(feature = "multiple_mca"))]
     #[schema(value_type = Option<PrimaryBusinessDetails>)]
     pub primary_business_details: Option<Vec<PrimaryBusinessDetails>>,
+    ///Will be used to expire client secret after certain amount of time to be supplied in seconds
+    ///(900) for 15 mins
```

Comment:
add the example as, you can also provide min and max value. Add punctuation marks
`#[schema(example = 900)]`

Reply:
also provide a default value